content/terms_and_definitions/terms_and_definitions.adoc

```diff
@@ -6,7 +6,86 @@ ifndef::include-only-once[]
 include::{root-path}_config.adoc[]
 endif::[]
 
-TODO: Add terms and definitions
+3D Asset::
+A 3D model. containing the 3D geometry, combined with assigned material properties and specific meta data.
 
-Static road network::
-Collection of connected roads enriched by static objects that do not change during runtime of a simulation.
+3D Model::
+A (mathematical) geometrical, coordinate-based representation of the surface of an object.
+This excludes physical material properties. For a combination of 3D geometry and physical material properties, see 3D Asset.
+
+3D Environment::
+A 3D representation of the surrounding of a simulated sensor.
+The 3D environment can consist of multiple 3D assets.
+
+Co-Simulation::
+Simulating multiple independent models connected with interfaces.
```

Comment:
Simulation of multiple independent models running simultaneously and connected with interfaces.

Reply:
The models are not necessarily running simultaneously. In many cases they run sequentially every time step. So I would remove "running simultaneously".

Reply:
done